app/components/Dropdown/AllowedClickInside.js

```diff
@@ -51,7 +51,7 @@ const Dropdown = ({ children, isOpen, target, toggle, isArrowed }) => (
       )}
     </TargetButton>
 
-    {isOpen ? <MenuStyled>{children}</MenuStyled> : null}
+    <MenuStyled isOpen={isOpen}>{children}</MenuStyled>
```

Comment:
the previous version is better, because dropdown content is not rendered if the dropdown is closed

```
{isOpen && <MenuStyled>{children}</MenuStyled>}
```